cobc/cobc.h

```diff
@@ -237,6 +237,14 @@ enum cb_dpc_in_data_options {
 	CB_DPC_IN_ALL
 };
 
+/* Character set used to decode EBCDIC symbolic characters in alphanumeric
+   literals */
+enum cb_ebcdic_literal_charset {
```

Comment:
I somehow think that if we add this, then there should be more than only those two sets... Adding IBM default for US and Europe seems to be reasonable.
Note: I _guess_ the translation table was coded similar to MF initially.

Reply:
Am I right in assuming that to convert from, *eg*, CP037 (EBCDIC-US) would require a special handling for non-representable characters? At the beginning I just dealt with the few "total" conversions I could find: the one in GCOS7 documentation, and the one already in GnuCOBOL's source code.

Reply:
For timing reasons I would like to postpone the introduction of additional charsets, for which tedious research of proper documentations might be required — also a proper policy for handling partial conversions might be required (error? warn and use `space`…).
I also have the feeling that, going down that line further may lead to rather substantial reworks for parameterising the collating sequences (used in `SORT`, for instance).